haas/cli.py

```diff
@@ -89,7 +91,18 @@ def do_delete(url):
     return check_status_code(requests.delete(url))
 
 @cmd
-def serve():
+def serve(port):
+    isNotValidPort = False
+    if port.isdigit():
+        port = int(port)
+        if (port < MIN_PORT_NUMBER) or (port > MAX_PORT_NUMBER):
+            isNotValidPort = True
+    else:
+	isNotValidPort = True
```

Comment:
Check out the schema library (we're using it elsehwere in the code) --- I bet you can get this down to a (readable) one-liner.